src/relayclient/RelayProvider.ts

```diff
@@ -18,7 +18,7 @@ export interface BaseTransactionReceipt {
 
 export type JsonRpcCallback = (error: Error | null, result?: JsonRpcResponse) => void
 
-export default class RelayProvider implements HttpProvider {
+export class RelayProvider implements HttpProvider {
```

Comment:
Why not default?

Reply:
Because it is then imported after compiled to js as `Gsn = require('...').default` which is ugly. Now it is imported as `Gsn = require('...').RelayProvider`.